src/server/pps/server/api_server.go

```diff
@@ -2125,16 +2125,46 @@ func getExpectedNumWorkers(pipelineInfo *pps.PipelineInfo) (int, error) {
 func (a *apiServer) CreatePipelineV2(ctx context.Context, request *pps.CreatePipelineV2Request) (resp *pps.CreatePipelineV2Response, err error) {
 	metricsFn := metrics.ReportUserAction(ctx, a.reporter, "CreatePipelineV2")
 	defer func(start time.Time) { metricsFn(start, err) }(time.Now())
-
-	js, err := a.createPipeline(ctx, request)
-	if err != nil {
+	var js string
```

Comment:
I believe this whole section can be reverted. `a.createPipeline()` will internally call createPipelineInTransaction